### ISSUE TITLE
Not able to select a file on Cordova

### ISSUE THREAD

**Issue by suhaboncukcu** [devonbarrett/meteor-dropzone#14]
Hey there,

I've just deployed my app to my android device by running the command 

 meteor run android-device --mobile-server http://mydomain

Dropzonejs is working on every browser on my mobile phone I've tried however when I deploy my app as above, it just doesn't show me any choice for uploading. Do you have any idea why is this happening? 

Best,

**Comment by devonbarrett** [comment]
Hey, 

I can't think of any reason why it won't work. Can you provide more info, do you get any errors anywhere? Are you trying to render a Dropzone programatically or using the template helper?

**Comment by suhaboncukcu** [comment]
Hi,

I'm using the template helper. However, I've figured out that this is a Cordova problem which dropzone.js itself is also suffering since Cordova doesn't support file input tags as it should. So I'm trying to find out how can I achieve this with another kind of thing (there are image pickers for cordova ios but I've failed to find a plugin for android so far.)  

In the end, it is not directly about your meteor plugin if you are not thinking to fix this in your template helper and this issue can be closed. 

Many thanks for your attention.
